packages/react/src/components/validation/validation.tsx

```diff
@@ -0,0 +1,30 @@
+import { c, classy, m } from '@onfido/castor';
+import { Icon } from '@onfido/castor-react';
+import React from 'react';
+
+/**
+ * Intended to be used alongside field components.
+ *
+ * `Validation` uses an `Icon` that requires `Icons` (SVG sprite) to be included
+ * in your app.
+ *
+ * https://github.com/onfido/castor-icons#use-with-plain-code
+ */
+export const Validation = ({
+  showIcon,
+  children,
+  className,
+  ...restProps
+}: ValidationProps): JSX.Element => (
+  <div
+    {...restProps}
+    className={classy(c('validation'), m({ showIcon }), className)}
```

Comment:
I don't think `m({ showIcon })` is doing anything, is it?

Reply:
Good point, I added but realised no need to do anything with styling.